<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWidgetClass</class>
 <widget class="QWidget" name="MapWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MAPFORM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(15,15,15);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background:rgb(15,15,15);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="MapGLWidget" name="openGLWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_download">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(66, 66, 66);
border: 0px solid #dcdfe6;
padding: 3px;
color:rgb(214, 214, 214);
}

QPushButton:hover {
background-color: rgb(64, 53, 130);
color:rgb(250, 250, 250);
}

QPushButton:pressed, QPushButton:checked {
background-color: rgb(145, 131, 238);
color:rgb(250, 250, 250);
}

#button3 {
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>DOWNLOAD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_findPathX">
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    padding-top: 2px;
    padding-bottom: 2px;
	padding-left: 4px;
    padding-right: 15px;
  border:1px solid rgb(66,66,66);
 	color:rgb(250,250,250);
  	background: rgb(56,56,56);
	selection-color: rgb(208,208,208);
	selection-background-color: rgb(80, 80, 83);
	font-family: &quot;Microsoft Yahei&quot;;
	font-size: 10pt;
}

QSpinBox:hover {
  color:rgb(250,250,250);
  background: rgb(31,31,31);
   border:1px solid rgb(153,153,153);
}

</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_findPathY">
           <property name="styleSheet">
            <string notr="true">QSpinBox {
    padding-top: 2px;
    padding-bottom: 2px;
	padding-left: 4px;
    padding-right: 15px;
  border:1px solid rgb(66,66,66);
 	color:rgb(250,250,250);
  	background: rgb(56,56,56);
	selection-color: rgb(208,208,208);
	selection-background-color: rgb(80, 80, 83);
	font-family: &quot;Microsoft Yahei&quot;;
	font-size: 10pt;
}

QSpinBox:hover {
  color:rgb(250,250,250);
  background: rgb(31,31,31);
   border:1px solid rgb(153,153,153);
}

</string>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_findPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(66, 66, 66);
border: 0px solid #dcdfe6;
padding: 3px;
color:rgb(214, 214, 214);
}

QPushButton:hover {
background-color: rgb(64, 53, 130);
color:rgb(250, 250, 250);
}

QPushButton:pressed, QPushButton:checked {
background-color: rgb(145, 131, 238);
color:rgb(250, 250, 250);
}

#button3 {
border-radius: 20px;
}</string>
           </property>
           <property name="text">
            <string>ROUTE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_returnBase">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgb(66, 66, 66);
border: 0px solid #dcdfe6;
padding: 3px;
color:rgb(214, 214, 214);
}

QPushButton:hover {
background-color: rgb(64, 53, 130);
color:rgb(250, 250, 250);
}

QPushButton:pressed, QPushButton:checked {
background-color: rgb(145, 131, 238);
color:rgb(250, 250, 250);
}

#button3 {
border-radius: 20px;
}</string>
         </property>
         <property name="text">
          <string>LOGBACK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_NPCList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft JhengHei UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableWidget {
	color: rgb(214,214,214);
    background: rgb(31,31,31);
    border: 1px solid rgb(61,61,61);
    alternate-background-color: #525252;
    gridline-color: transparent;

}

QTableView QTableCornerButton::section {
	color: rgb(214,214,214);
    background: rgb(31,31,31);
    border: 0px solid #418bc9;/*边框*/
    border-radius:0px;/*边框圆角*/
    border-color: rgb(41, 139, 201);/*边框颜色*/
    font: bold 11pt;/*字体大小*/
    padding:12px 0 0 10px;/*内边距*/
 }

QTableWidget::item {
    background: rgb(31,31,31);
}

QTableWidget::item:selected {
    background:rgb(61,61,61); /* qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #484848, stop:1 #383838);*/
}

QTableWidget::item:hover {
    background:rgb(61,61,61);
}

QHeaderView {
    text-align: center;
    background: rgb(31,31,31);
    padding: 0px;
    margin: 0px;
    color: rgb(250,250,250);
    border: 0px solid rgb(61,61,61);

}

QHeaderView::section::vertical {
    text-align: center;
    background: rgb(31,31,31);
    padding: 3px;
    margin: 0px;
    color: rgb(250,250,250);
    border: 0px solid rgb(61,61,61);
    border-left-width: 0;
	border-bottom-width: 1;
	border-right-width: 1;
}

QHeaderView::section::horizontal{
    text-align: center;
    background: rgb(31,31,31);
    padding: 3px;
    margin: 0px;
    color: rgb(250,250,250);
    border: 0px solid rgb(61,61,61);
    border-left-width: 0;
	border-bottom-width: 1;
	border-right-width: 1;
}

QScrollBar:vertical {
    background: rgb(46,46,46);
}

QScrollBar::handle:vertical {
    background: rgb(71,71,71);
  	border: 3px solid  rgb(46,46,46);
}

QScrollBar::handle:hover:vertical,
QScrollBar::handle:pressed:vertical {
    background: rgb(153,153,153);
}

QScrollBar::sub-page:vertical {
    background: 444444;
}

QScrollBar::add-page:vertical {
    background: 5B5B5B;
}

QScrollBar::add-line:vertical {
    background: none;
}

QScrollBar::sub-line:vertical {
    background: none;
}

QScrollBar:horizontal {
    background: rgb(46,46,46);
}

QScrollBar::handle:horizontal {
    background: rgb(71,71,71);
  	border: 3px solid  rgb(46,46,46);
}

QScrollBar::handle:hover:horizontal,
QScrollBar::handle:pressed:horizontal {
    background: rgb(153,153,153);
}

QScrollBar::sub-page:horizontal {
    background: 444444;
}

QScrollBar::add-page:horizontal {
    background: 5B5B5B;
}

QScrollBar::add-line:horizontal {
    background: none;
}

QScrollBar::sub-line:horizontal {
    background: none;
}

QToolTip{
border-style:none; 
background-color: rgb(57, 58, 60);
color: rgb(208, 208, 208);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>22</number>
         </attribute>
         <column>
          <property name="text">
           <string>NAME</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>LOCATE</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">mapglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
